packages/web-components/src/components/ic-classification-banner/ic-classification-banner.tsx

```diff
@@ -37,7 +37,19 @@ export class ClassificationBanner {
   @Prop() upTo?: boolean = false;
 
   render() {
-    const { classification, inline, country, upTo, additionalSelectors } = this;
+    const { inline, upTo } = this;
+
+    // In case of unrecognized props, fallback to default
+    let { country, additionalSelectors, classification } = this;
+    if (!country || country === undefined) country = "uk";
+    if (!additionalSelectors || additionalSelectors === undefined)
+      additionalSelectors = "";
+    if (
+      !classification ||
+      classification === undefined ||
+      (classification && !classificationText[classification])
+    )
+      classification = "default";
```

Comment:
lmk if you want braces here or extract the conditionals to statements. Prettier indentation made it a bit unreadable.